package.json

```diff
@@ -10,13 +10,18 @@
     }, {
       "name": "AB Zainuddin",
       "email": "burhan@codeyellow.nl"
+    }, {
+      "name": "Steve James",
     }
   ],
   "repository": {
     "type": "git",
     "url": "https://github.com/ehynds/jquery-ui-multiselect-widget"
   },
   "main": "src/jquery.multiselect.js",
-  "dependencies": {},
+  "dependencies": {
+    "jquery": "^1.8.0",
+    "jquery-ui": "^1.8.0"
```

Comment:
I think it's actually jquery-ui 1.11.0 since I started using the `instance` method from the widget factory.

Reply:
Will fix.